apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/R4PatientResourceProvider.java

```diff
@@ -0,0 +1,838 @@
+package gov.cms.bfd.server.war.r4.providers;
```

Comment:
Same as above, I would consider removing version indicators from the filenames.

Reply:
See above.